mcux/drivers/kinetis/CMakeLists.txt

```diff
@@ -15,6 +15,7 @@ zephyr_library_sources_ifdef(CONFIG_ADC_MCUX_ADC12	fsl_adc12.c)
 zephyr_library_sources_ifdef(CONFIG_ADC_MCUX_ADC16	fsl_adc16.c)
 zephyr_library_sources_ifdef(CONFIG_CAN_MCUX_FLEXCAN	fsl_flexcan.c)
 zephyr_library_sources_ifdef(CONFIG_COUNTER_MCUX_LPTMR	fsl_lptmr.c)
+zephyr_library_sources_ifdef(CONFIG_COUNTER_MCUX_PIT    fsl_pit.c)
```

Comment:
Sorry for the nitpick, but please replace the spaces with a tab to match the rest of the file.

Reply:
OK